packages/brand-positioning-block/src/settings.ts

```diff
@@ -1,18 +1,438 @@
+                            defaultValue: 'X Axis Right',
+                        },
+                        {
+                            id: 'yAxisBottomLabel',
+                            type: 'input',
+                            placeholder: 'Y Axis Bottom Label',
+                            defaultValue: 'Y Axis Bottom Label',
+                        },
+                    ],
+                },
+            ],
+        },
+    ],
+    style: [
+        {
+            id: 'board',
+            type: 'sectionHeading',
+            label: 'Board',
+            blocks: [
+                {
+                    id: 'boardAxisLines',
+                    type: 'multiInput',
+                    label: 'Axis lines',
+                    lastItemFullWidth: true,
+                    blocks: [
+                        {
+                            id: 'boardAxisLinesStyle',
+                            type: 'dropdown',
+                            defaultValue: 'solid',
+                            size: DropdownSize.Small,
+                            choices: [
+                                {
+                                    value: 'dotted',
+                                    label: 'Dotted',
+                                },
+                                {
+                                    value: 'dashed',
+                                    label: 'Dashed',
+                                },
+                                {
+                                    value: 'solid',
+                                    label: 'Solid',
+                                },
+                            ],
+                        },
+                        {
+                            id: 'boardAxisLinesWidth',
+                            type: 'input',
+                            defaultValue: '1px',
+                            placeholder: '1px',
+                            rules: [numericalOrPixelRule],
+                            onChange: (bundle) => {
+                                appendUnit(bundle, 'boardAxisLinesWidth');
+                            },
+                        },
+                        {
+                            id: 'borderColor',
+                            type: 'colorInput',
+                            defaultValue: BOARD_AXIS_LINES_COLOR_DEFAULT_VALUE,
+                        },
+                    ],
+                    layout: MultiInputLayout.Columns,
+                },
+                {
+                    id: 'boardAxisLabelsColor',
+                    type: 'colorInput',
+                    label: 'Axis labels',
+                    defaultValue: BOARD_AXIS_LABELS_COLOR_DEFAULT_VALUE,
+                },
+                {
+                    id: 'boardBackgroundColor',
+                    type: 'colorInput',
+                    label: 'Background',
+                    defaultValue: BOARD_BACKGROUND_COLOR_DEFAULT_VALUE,
+                },
+                {
```

Comment:
For borders and border radiuses you can reuse `getBorderRadiusSettings, getBorderSettings` from the `@frontify/guideline-blocks-shared` package